metis-dereference/metis-dereference-client/src/test/resources/client.properties.example

```diff
@@ -0,0 +1 @@
+host.url=
```

Comment:
You copied this file to cclient.properties in the main resource folder. I don't think cclient.properties should be there, this file is sufficient I think.

Reply:
This file has been removed.